lib/tables.dart

```diff
@@ -420,10 +420,25 @@ class Table<T> extends Object with SetStateMixin {
   /// and not necessarily for rows[] since it may be being rendered in reverse.
   /// This way, +1 will always move down the visible table.
   @visibleForTesting
-  void selectByIndex(int newIndex) {
+  void selectByIndex(int newIndex,
```

Comment:
For the dartdoc, reference the docs for HtmlElement.scrollTo? Or, mention that valid values for scrollBehavior are `auto` and `smooth`.